index.html

```diff
@@ -16,7 +16,7 @@
     </header>
     <section class="section1">
         <h1 class="header"><span>Hey there.
-                <br>I’m Alex.</span>
+                <br>I’m Ian.</span>
             <span class="text_color">I’m a software developer</span>
         </h1>
         <p class="paragraph"> I can help you build a product , <br>feature or website Look through <br>some of my work
```

Comment:
- [ ] Your UI looks great but there are some issues you need to fix before moving to the next milestone.

-  ![Screenshot from 2023-06-13 15-21-04](https://github.com/NgachaIan/Personal_Portfolio/assets/107632780/bd5b9d21-443b-46eb-b0fb-b77cac8b6b61)
- [ ] Following the best practices, you can only have one `h1` in a page and that's the main title, so this should be an `h4` or a p tag, I'll suggest you update this.  I'll also suggest you add margin left and wright as there are spaces in both sides from the Figma design.:+1: :100: 

-  ![Screenshot from 2023-06-13 15-23-08](https://github.com/NgachaIan/Personal_Portfolio/assets/107632780/560746ec-a421-4b85-888a-ae069eb1986b)
- [ ] Currently after filling up the form and clicking on the button, nothing happens, this is because you gave the button the wrong type, I'll suggest you change to type to `submit` which should be the correct type for the form submit button. :+1: 

- ![Screenshot from 2023-06-13 15-25-09](https://github.com/NgachaIan/Personal_Portfolio/assets/107632780/70e74dae-6d14-4140-ab97-0ad3a165cc3c)
- [ ] Just as I suggested above, you are meant to have only one `h1`, I'll suggest you change this to the appropriate tag. :+1: :smile: